website/docs/r/vdc_group.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vdc_group"
+sidebar_current: "docs-vcd-resource-vdc-group"
+description: |-
+  Provides a VDC group resource.
+---
+
+# vcd\_vdc\_group
+Supported in provider *v3.5+* and VCD 10.2+.
+
+Provides a resource to manage NSX-T VDC groups.
+
+~> Only `System Administrator` and `Org Users` with rights `View VDC Group`, `Configure VDC Group`, `vDC Group: Configure Logging`, `Organization vDC Distributed Firewall: Enable/Disable` can manage VDC groups using this resource.
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "startVdc" {
+  name = "existingVdc"
+}
+
+data "vcd_org_vdc" "additionalVdc" {
+  name = "oneMoreVdc"
+}
+
+resource "vcd_vdc_group" "new-vdc-group" {
+  org                   = "myOrg"
+  name                  = "newVdcGroup"
+  description           = "my description"
+  starting_vdc_id       = data.vcd_org_vdc.startVdc.id
+  participating_vdc_ids = [data.vcd_org_vdc.startVdc.id, data.vcd_org_vdc.additionalVdc.id]
+  dfw_enabled           = true
+  default_policy_status = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organizations
+* `name` - (Required)  - The name for VDC group
```

Comment:
We don't use ` - ` separator after `(Required)` or `(Optional)`. Please fix all occurrences in this PR.

```suggestion
* `name` - (Required) The name for VDC group
```

Reply:
changed